app/components/Modals/SendModal/SendModal.jsx

```diff
@@ -85,15 +89,27 @@ export default class SendModal extends Component<Props, State> {
           onCancel={this.handleCancelTransaction}
           onAddRecipient={this.handleAddRecipient}
           onDelete={this.handleDeleteEntry}
+          onUpdatePriorityFee={this.handleUpdatePriorityFee}
+          toggleCollapsedFeeInfo={this.handleToggleCollapsedFeeInfo}
           {...this.state} />
       )
     }
   }
 
+  handleUpdatePriorityFee = (priorityFee: string) => this.setState({ priorityFee })
+
   handleAddRecipient = () => {
     this.setState({ display: DISPLAY_MODES.ADD_RECIPIENT })
   }
 
+  determineModalHeight = () => {
+    const { display, priorityFeeCollapsed } = this.state
+    if (display === DISPLAY_MODES.ADD_RECIPIENT) {
+      return '350px'
+    } else if (priorityFeeCollapsed) return '350px'
+    return '485px'
```

Comment:
shouldn't this be: 

`return display === DISPLAY_MODES.ADD_RECIPIENT || priorityFeeCollapsed ? '350px' : '485px'` ? (you return the same value in 2 conditions)

also, would be best if these are constants